docs/user_manual/customize_metric.rst

```diff
@@ -149,7 +149,8 @@ Understanding Call Types
 +--------------------+-------------------------------------------------------------+
 | `pairwise_gt_y`    | Subsequent model's output first, then base model's output   |
 +--------------------+-------------------------------------------------------------+ 
-
+| `y`                | Only the output is used, the metric has an internal dataset |
++--------------------+-------------------------------------------------------------+ 
```

Comment:
@begumcig I think the behaviour with `single` vs `pairwise` might need to be updated too here?

https://docs.pruna.ai/en/stable/docs_pruna/user_manual/evaluate.html#internal-call-types